app/controllers/kalibro_configurations_controller.rb

```diff
@@ -46,9 +46,29 @@ def exists
   end
 
   def metric_configurations
-    if set_kalibro_configuration
-      respond_to do |format|
-        format.json { render json: {metric_configurations: @kalibro_configuration.metric_configurations} }
+    return unless set_kalibro_configuration
+    # TODO: refactor this code to a private method
```

Comment:
Because it wouldn't improve the main feature of this branch. It still seems necessary though.